Assets/Scripts/PlayerInventory.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/**
+    Klasa ekwipunku dla gracza.
+    Umożliwia używanie przedmiotów.
+**/
+public class PlayerInventory : Inventory {
+
+    public float fadeInScale = 25f;
+    public float fadeOutScale = 15f;
+    public Sprite emptySlotSprite;
+
+    public PlayerControlls player;
+    public Canvas canvas;
+
+    UnityEngine.UI.Image[] images;
+    int lastSelected = 0;
+    ItemObject[] equipped;
+    ItemObject[] picked;
+
+	// Use this for initialization
+	void Start () {
+        images = new UnityEngine.UI.Image[8];
+        if (player == null)
+        {
+            player = gameObject.GetComponent<PlayerControlls>();
+        }
+        if (canvas == null)
+        {
+            canvas = gameObject.transform.GetComponentInChildren<Canvas>();
+        }
+        equipped = new ItemObject[2];
+        picked = new ItemObject[7];
+        canvas.enabled = false;
+        {
+            int i = 0;
+            foreach (Transform child in canvas.transform)
+            {
+                images[i] = child.gameObject.GetComponent<UnityEngine.UI.Image>();
+                i++;
+            }
+        }
+	}
+	
+    public bool isInventoryOpen()
+    {
+        return Input.GetButton("Inventory_" + player.getPlayerNum());
+    }
+
+    void ShowInventory()
+    {
+        canvas.scaleFactor = Mathf.Lerp(canvas.scaleFactor, 1.0f, Time.deltaTime * fadeInScale);
+        if (canvas.scaleFactor > 0.2f)
+            canvas.enabled = true;
+        Vector3 aim = player.getLastTurn();
+        Vector2 aim2d = Vector2.zero;
+        aim2d.Set(aim.x, aim.z);
+        if (aim2d.SqrMagnitude() > 0.5)
```

Comment:
In general it was only used as a kind of failsafe mechanism. It is to be redeclared, so probably wont be needed.